### PR TITLE
Feature/gh repo contents

### LINKED ISSUE #25: Add cli to read a given github repository file
Using the cli it should be possible to fetch 1 or many files from a given repository.

e.g.:

```bash
bin/tabia github files --repo philips-labs/tabia --files README.md --files LICENSE
```

### PR DESCRIPTION
**Pull request by marcofranssen**
Adds a new cli to fetch a file from a github repository.

```bash
bin/tabia github contents --repo philips-labs/tabia --file README.md --output downloads/tabia/README.md
```

It supports folders to be defined in the output path.

resolves #25

### CODE REVIEW

File: lib/github/contents_test.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+func TestDownloadContents(t *testing.T) {
+	assert := assert.New(t)
+	gh := github.NewClientWithTokenAuth(os.Getenv("TABIA_GITHUB_TOKEN"))
+	contents, err := gh.DownloadContents(context.Background(), "philips-labs", "tabia", "README.md")
+	if assert.NoError(err) {
+		readme, _ := ioutil.ReadFile("../../README.md")
+		assert.NotEmpty(contents)
+		assert.Equal(string(readme), string(contents))
+	}
+}
```
Comment (bartgolsteijn) [inline lib/github/contents_test.go:23]: Do you want to test error handling, e.g. non-existent file, as well?

File: lib/github/repositories.go [hunk 2]
```diff
@@ -109,30 +111,31 @@ func (c *Client) FetchOrganziationRepositories(ctx context.Context, owner string
...
 		return nil, err
 	}
 	return Map(repositories, privateRepos)
 }
 
-func (c *Client) FetchRestRepositories(owner, repoType string) ([]RestRepo, error) {
-	resp, err := c.httpClient.Get(fmt.Sprintf("https://api.github.com/orgs/%s/repos?type=%s", owner, repoType))
+func (c *Client) FetchRestRepositories(ctx context.Context, owner, repoType string) ([]*github.Repository, error) {
+	repos, resp, err := c.restClient.Repositories.ListByOrg(ctx, owner, &github.RepositoryListByOrgOptions{Type: repoType})
+	// resp, err := c.httpClient.Get(fmt.Sprintf("https://api.github.com/orgs/%s/repos?type=%s", owner, repoType))
```
Comment (bartgolsteijn) [inline lib/github/repositories.go:123]: Remove this line?

File: lib/github/repositories.go [hunk 3]
```diff
@@ -109,30 +111,31 @@ func (c *Client) FetchOrganziationRepositories(ctx context.Context, owner string
...
 	if err != nil {
 		return nil, err
 	}
 	defer resp.Body.Close()
+	return repos, nil
+	// var privateRepos []RestRepo
```
Comment (bartgolsteijn) [inline lib/github/repositories.go:129]: Remove commented-out lines?

File: lib/github/contents_test.go (hunk 1 above)
Comment (marcofranssen) [reply lib/github/contents_test.go:23]: Good point. Let me add that.

File: lib/github/repositories.go (hunk 2 above)
Comment (marcofranssen) [reply lib/github/repositories.go:123]: Will do 👍

**Pull request merged**